### PR TITLE
Mount Baby Cost Calculator

### PR DESCRIPTION
**Pull request by Tr4pSt3R**

### CODE REVIEW

**Comment by benlovell** [comment]
Looks good to me. But that history needs squashing down to something sane. :+1:

File: spec/requests/tool_integration/baby_cost_calculator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+RSpec.describe ToolMountPoint::BabyCostCalculator, type: :request, features: [:baby_cost_calculator] do 
+  %W(
+    /en/tools/#{ToolMountPoint::BabyCostCalculator::EN_ID}
```
Comment (asmega) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: may as well stick in the Welsh version

**Comment by darokel** [comment]
:+1: As per the comments above

File: spec/requests/tool_integration/baby_cost_calculator_spec.rb (hunk 1 above)
Comment (Tr4pSt3R) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: Hey @asmega yeah that's done now. Thanks Phil

**Comment by benlovell** [comment]
Squashy squash and :ship:

**Comment by asmega** [comment]
:+1:

**Comment by benlovell** [comment]
@Tr4pSt3R is there any logic to the prefixes you're applying to your commit messages? I don't think it's necessary.

**Comment by Tr4pSt3R** [comment]
@benlovell I would need your help in squashing these messages.

**Pull request merged**